examples/xml-from-record-conversion/xml_from_record_conversion.bal

```diff
@@ -0,0 +1,40 @@
+import ballerina/io;
+import ballerina/xmldata;
+
+// Defines a record type with annotations.
+@xmldata:Namespace {
+    prefix: "ns",
+    uri: "http://sdf.com"
+}
+type Invoice record {
+    int id;
+    Item[] items;
+    @xmldata:Attribute
+    string 'xmlns = "example.com";
+    @xmldata:Attribute
+    string attr?;
+};
+
+@xmldata:Namespace {
+    uri: "http://example1.com"
+}
```

Comment:
Shall we not add namespace annotation for both record types?

Reply:
No need to add a namespace annotation for both record types. If we want to add `xmlns` with a different URL in the child sequence, we can add it in the child type like this.